layer/runtime/runtime_layer.cc

```diff
@@ -483,3 +493,65 @@ SPL_LAYER_ENTRY_POINT SPL_RUNTIME_LAYER_FUNC(PFN_vkVoidFunction,
   assert(next_get_proc_addr && next_get_proc_addr != VK_NULL_HANDLE);
   return next_get_proc_addr(instance, name);
 }
+
+#if defined(EXPOSE_LAYER_INTERFACE_VERSION_0)
+
+#define LAYER_NAME_FUNCTION_CONCAT(layername, func) layername##func
+#define LAYER_NAME_FUNCTION_CONCAT_EXPAND(layername, func) \
+  LAYER_NAME_FUNCTION_CONCAT(layername, func)
+#define LAYER_NAME_FUNCTION(func) \
+  LAYER_NAME_FUNCTION_CONCAT_EXPAND(LAYER_NAME, func)
+
+// Exposes the layer interface version 0's GetInstanceProcAddr
+SPL_LAYER_ENTRY_POINT VKAPI_ATTR PFN_vkVoidFunction VKAPI_CALL
+LAYER_NAME_FUNCTION(GetInstanceProcAddr)(VkInstance instance,
```

Comment:
What do you think about one more macro to define an entry point function with a given name and type? I would see it used like this:

```c++
SPL_RUNTIME_LAYER_INTERFACE(PFN_vkvoidFunvtion, GetInstanceProcAddr)((VkInstance instance,
```

Reply:
We have something similar for the existing functions above `SPL_RUNTIME_LAYER_FUNC`

Reply:
I added a new macro `SPL_RUNTIME_LAYER_INTERFACE` that uses the same arguments as `SPL_RUNTIME_LAYER_FUNC` does (i.e. return type, func name and args).
I use the macro for the two entry points that require the layer name prepended.